### PR TITLE
doc: new guide to deploy on Cloud Run

### LINKED ISSUE #50: Implement example deploying to Cloud Run

### LINKED ISSUE #226: Create a HOWTO guide for deploying a Pub/Sub function to Cloud Run
Should include a description of how to manually test this function and examine its logs.

### PR DESCRIPTION
**Pull request by coryan**
Also refactored the existing guide to create containers to make both
look similar.

Fixes #226. Part of the work for #202

### CODE REVIEW

File: examples/site/howto_create_container/README.md [hunk 1]
```diff
@@ -1,41 +1,117 @@
...
+  response.set_payload(greeting());
+  response.set_header("content-type", "text/plain");
+  return response;
+}
+```
+
+## Getting the code for this example
+
+This example is included in the Functions Framework for C++
+[source code repository][repository], download this code as usual:
```
Comment (devbww) [inline examples/site/howto_create_container/README.md:45]: s/, download/. Download/

File: examples/site/howto_create_container/README.md [hunk 2]
```diff
@@ -1,41 +1,117 @@
...
+
+## Cleanup
+
+Stop the background container:
+
+```shell
+docker kill "${ID}"
+```
+
+And delete the local image
```
Comment (devbww) [inline examples/site/howto_create_container/README.md:113]: s/$/:/

File: examples/site/howto_deploy_to_cloud_run/README.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  response.set_payload(greeting());
+  response.set_header("content-type", "text/plain");
+  return response;
+}
+```
+
+## Getting the code for this example
+
+This example is included in the Functions Framework for C++
+[source code repository][repository], download this code as usual:
```
Comment (devbww) [inline examples/site/howto_deploy_to_cloud_run/README.md:52]: s/, download/. Download/

File: examples/site/howto_deploy_to_cloud_run/README.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+docker build -t gcf-cpp-runtime --target gcf-cpp-runtime -f build_scripts/Dockerfile build_scripts
+pack create-builder gcf-cpp-builder:bionic --config pack/builder.toml
+pack trust-builder gcf-cpp-builder:bionic
+pack set-default-builder gcf-cpp-builder:bionic
+```
+
+## Building a Docker image
+
+Set the `GOOGLE_CLOUD_PROJECT` shell variable to the project id of your GCP
+project, and create a docker image with your function.
```
Comment (devbww) [inline examples/site/howto_deploy_to_cloud_run/README.md:84]: s/\./:/

File: examples/site/howto_deploy_to_cloud_run/README.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+Push this new container image to Google Container Registry:
+
+```shell
+docker push "gcr.io/${GOOGLE_CLOUD_PROJECT}/gcf-cpp-hello-world-http:latest"
+```
+
+## Deploy to Cloud Run
+
+To deploy this image in Cloud Run use this command. You need to select
+a Cloud Run region for your deployment, we will use `us-central1` in this
```
Comment (devbww) [inline examples/site/howto_deploy_to_cloud_run/README.md:105]: s/, we/. We/

**Review by devbww — APPROVED** [review]

File: examples/site/howto_deploy_to_cloud_run/README.md (hunk 5 above)
Comment (coryan) [reply examples/site/howto_deploy_to_cloud_run/README.md]: Fixed.

File: examples/site/howto_deploy_to_cloud_run/README.md (hunk 4 above)
Comment (coryan) [reply examples/site/howto_deploy_to_cloud_run/README.md]: Fixed.

File: examples/site/howto_deploy_to_cloud_run/README.md (hunk 3 above)
Comment (coryan) [reply examples/site/howto_deploy_to_cloud_run/README.md]: Done.

File: examples/site/howto_create_container/README.md (hunk 2 above)
Comment (coryan) [reply examples/site/howto_create_container/README.md]: Done.

File: examples/site/howto_create_container/README.md (hunk 1 above)
Comment (coryan) [reply examples/site/howto_create_container/README.md]: Done.

**Pull request merged**